docker-bake.hcl

```
@@ -59,7 +59,7 @@ target "buildkit" {
   inherits = ["mainline"]
   args = {
     BINARY_PREFIX = "buildkit-"
-    QEMU_PATCHES = "${QEMU_PATCHES},buildkit-direct-execve-v8.1"
+    QEMU_PATCHES = "${QEMU_PATCHES},buildkit-direct-execve-v9.2.0"
```

Needs to rename folder `patches/buildkit-direct-execve-v9.1.2` to `patches/buildkit-direct-execve-v9.2` and use major minor:

```suggestion
    QEMU_PATCHES = "${QEMU_PATCHES},buildkit-direct-execve-v9.2"
```

---

Thx :)